BeSpoke/Models/Note.swift

```diff
@@ -6,12 +6,70 @@
 //
 
 import Foundation
+import FirebaseFirestoreSwift
+import FirebaseFirestore
 import MapKit
 
+// encode/decode functionality manually implemeted to
+// allow using DocumentID as internal ID
```

Comment:
We can maybe come up with a way to generalize this instead of having the same-ish methods in each model in the future (assuming they are the same-ish) but not a problem for now

Reply:
How very object oriented of you, not sure how inheritance works here but added this here #10.